### ISSUE TITLE
Add new vertices to the mesh

### ISSUE THREAD

**Issue by TheShasa** [mikedh/trimesh#2226]
I have a mesh and points which lies on surfaces on this mesh. How can I triangulate the mesh to consist this new vertices? Basically new faces and vertices should be added. 

Thank you.

**Comment by Kiord** [comment]
I don't think that Trimesh has this kind of feature yet. Maybe you can try to remove the affected (and nearby) faces  and retriangulate them in a second step with [scipy's Delaunay triangulation](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.Delaunay.html).
